phpcs.xml

```diff
@@ -1,21 +1,21 @@
 ﻿<?xml version="1.0"?>
-<ruleset name="Laminas coding standard">
-    <!-- inherit rules from: -->
-    <rule ref="PSR2"/>
-    <rule ref="Generic.Arrays.DisallowLongArraySyntax"/>
-    <rule ref="Generic.Formatting.SpaceAfterNot"/>
-    <rule ref="Squiz.WhiteSpace.OperatorSpacing">
-        <properties>
-            <property name="ignoreNewlines" value="true"/>
-        </properties>
-    </rule>
-    <rule ref="Squiz.WhiteSpace.SuperfluousWhitespace">
-        <properties>
-            <property name="ignoreBlankLines" value="false"/>
-        </properties>
-    </rule>
+<ruleset xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:noNamespaceSchemaLocation="vendor/squizlabs/php_codesniffer/phpcs.xsd">
+
+    <arg name="basepath" value="."/>
+    <arg name="cache" value=".phpcs-cache"/>
+    <arg name="colors"/>
+    <arg name="extensions" value="php"/>
+    <arg name="parallel" value="80"/>
+
+    <!-- Show progress -->
+    <arg value="ps"/>
 
     <!-- Paths to check -->
     <file>src</file>
     <file>test</file>
+    <exclude-pattern>src/Controller/MiddlewareController.php</exclude-pattern>
```

Comment:
If this file is not exclude it crash

Reply:
Just checked out this PR and phpcs is fine without it for me and reporting a bunch of issues. Can you try another `composer update` ensuring you do that on at least PHP 8.1?